### ISSUE TITLE
sample page is broken

### ISSUE THREAD

**Issue by williandandrade** [oblador/angular-parallax#24]
I'm using Chrome 45.0.2454.85 on Windows 10.

```
TypeError: Cannot read property 'compile' of undefined
    at http://ajax.googleapis.com/ajax/libs/angularjs/1.2.19/angular.min.js:44:276
    at Array.forEach (native)
    at q (http://ajax.googleapis.com/ajax/libs/angularjs/1.2.19/angular.min.js:7:280)
    at Object.<anonymous> (http://ajax.googleapis.com/ajax/libs/angularjs/1.2.19/angular.min.js:44:209)
    at Object.d [as invoke] (http://ajax.googleapis.com/ajax/libs/angularjs/1.2.19/angular.min.js:35:36)
    at http://ajax.googleapis.com/ajax/libs/angularjs/1.2.19/angular.min.js:36:288
    at Object.c [as get] (http://ajax.googleapis.com/ajax/libs/angularjs/1.2.19/angular.min.js:34:305)
    at ba (http://ajax.googleapis.com/ajax/libs/angularjs/1.2.19/angular.min.js:58:283)
    at ca (http://ajax.googleapis.com/ajax/libs/angularjs/1.2.19/angular.min.js:49:263)
    at N (http://ajax.googleapis.com/ajax/libs/angularjs/1.2.19/angular.min.js:47:351)
```

http://oblador.github.io/angular-parallax/

**Comment by oblador** [comment]
Cannot reproduce with 45.0.2454.85 on OSX. Try again in incognito mode.

**Comment by williandandrade** [comment]
![capture](https://cloud.githubusercontent.com/assets/838425/9745311/5b554422-5647-11e5-99f2-c788030a5e74.PNG)

Same problem. I can test later on my osx machine.

**Comment by williandandrade** [comment]
@oblador, works on OSX!

Probably is a specific problem with Chrome running on windows.

**Comment by oblador** [comment]
Very strange, don't have access to a Windows comp atm. Can you please investigate? :-)

**Comment by williandandrade** [comment]
I can try. I downloaded the project and access the page in the "example" directory and the error didn't happen. I found differences between published example code and example in the source code, perhaps an update to the "gh-pages" solve the problem.

**Comment by ettiennegous** [comment]
Can confirm the version of the JS file on Github pages is not working.
I made a CodePen to illustrate here
https://codepen.io/ettman/pen/wPpVVX

If I just copy the minified JS file from the repo into my pen rather than ref the file from github pages it starts working.

TL;DR
This file is stuffed
http://oblador.github.io/angular-parallax/angular-parallax.min.js
